features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/PrivacyOverrideExtensions.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay
+
+import android.view.View
+import com.datadog.android.sessionreplay.internal.PrivacyOverrideManager
+
+/**
+ * Allows setting a view to be "hidden" in the hierarchy in Session Replay.
+ * When hidden the view will be replaced with a placeholder in the replay and
+ * no attempt will be made to record it's children.
+ *
+ * @param hide pass `true` to hide the view, or `false` to remove the override
+ */
+fun View.ddSessionReplayOverrideHidden(hide: Boolean) {
```

Comment:
I think in the RFC, the API should be able to pass the sdk core inside? 
```
view.ddSessionReplayOverrideHidden(sdkInstance, hide = true)
```
It can avoid to make `PrivacyOverrideManager` as an singleton object, and will be easier to make the test I think

Reply:
Yes, the source of this rfc change was a discussion that I had today with xav. I'm in the process of making changes to the code to support multiple instances